demo/java/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+    maven {
+        url 'https://s01.oss.sonatype.org/content/repositories/aipicovoice-1034/'
+    }
+}
+
+sourceSets {
+    main {
+        java {
+            srcDirs = ['src']
+        }
+    }
+}
+
+dependencies {
+    implementation 'ai.picovoice:porcupine-java:1.9.0'
+    implementation 'commons-cli:commons-cli:1.4'
+}
+
+jar {
```

Comment:
this should mirror the file demo, but the setup differs

Reply:
only way to disable the main JAR build that I could find was to overwrite it with another operation. This prevents a useless jar from cluttering up the build directory.